views/default/panels/router/detail.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/* @var $model  yii\debug\models\Router */
+
+use \yii\helpers\Html;
+
+echo '<h1>Routing<small>' . Yii::t('yii', '{n, plural, =0{} =1{ - Tested {n} rule} other{ - Tested {n} rules}} {m, plural, =0{} other{before match}}', ['n' => $model->count, 'm' => (int)$model->hasMatch]) . '</small></h1>';
```

Comment:
There's no need to use `Yii::t` if you need formatting only. Use `Yii::$app->i18n->format('{n, plural, =0{} =1{ - Tested {n} rule} other{ - Tested {n} rules}} {m, plural, =0{} other{before match}}', ['n' => $model->count, 'm' => (int)$model->hasMatch], 'en')`.